OpenRA.Mods.Common/Traits/ProximityCapturable.cs

```diff
@@ -9,24 +9,44 @@
 #endregion
 
 using System.Collections.Generic;
+using System.Drawing;
 using System.Linq;
 using OpenRA.Effects;
+using OpenRA.Graphics;
+using OpenRA.Mods.Common.Graphics;
 using OpenRA.Traits;
 
 namespace OpenRA.Mods.Common.Traits
 {
+	public enum RangeCircleVisibility { Never, Always, WhenSelected }
```

Comment:
Fixing `WithRangeCircle` makes sense. I never quite liked its limitations. Updated.

@abcdefg30: I guess I'll have to remove your :+1: for that, sorry.